span.go

```diff
@@ -58,3 +62,30 @@ type Span interface {
 	// TraceContext.
 	AddToGoContext(goCtx context.Context) (Span, context.Context)
 }
+
+// See Span.Log(). Every LogData instance should specify at least one of Event
+// and/or Payload.
+type LogData struct {
+	// The timestamp of the log record; if set to the default value (the unix
+	// epoch), implementations should use time.Now() implicitly.
+	Timestamp time.Time
```

Comment:
So here we are saying the boundary, not the unit. Maybe make docs clear if that's the case. Ex not necessarily microseconds if that's the intent. If Timestamp already includes a unit (in go), maybe it is implied.